test/reductions/vass/reductions-perf.good

```diff
@@ -0,0 +1,6 @@
+4 locale(s)   dist: Block  sizes: 4000000 16384 16384 15625  reps: 10
```

Comment:
This file is used when CHPL_COMM != none, and also not doing performance testing.